server/main.py

```diff
@@ -31,14 +33,40 @@ class Payload(BaseModel):
     tvib: Optional[float] = None
     trot: Optional[float] = None
     path_length: float
-    simulate_slit: Optional[int] = None
-    mode: Literal["absorbance", "transmittance_noslit", "radiance_noslit", "transmittance", "radiance"]
+    simulate_slit: int
+    use_simulate_slit:bool
+    mode: Literal["absorbance", "transmittance_noslit", "radiance_noslit"]
     database: Literal["hitran", "geisa"]
-    use_simulate_slit: bool = False
+
+def calculate_spectrum(payload):
+    print(payload)
+    spectrum = radis.calc_spectrum(
+            payload.min_wavenumber_range,
+            payload.max_wavenumber_range,
+            molecule=[species.molecule for species in payload.species],
+            mole_fraction={
+                species.molecule: species.mole_fraction for species in payload.species
+            },
+            # TODO: Hard-coding "1,2,3" as the isotopologue for the time-being
+            isotope={species.molecule: "1,2,3" for species in payload.species},
+            pressure=payload.pressure,
+            Tgas=payload.tgas,
+            Tvib=payload.tvib,
+            Trot=payload.trot,
+            path_length=payload.path_length,
+            export_lines=False,
+            wstep="auto",
+            databank=payload.database,
+            use_cached=True,
+            local_folder="~/radis_spectra"
+        )
+    return spectrum
+    
+    
 
 
 @app.post("/calculate-spectrum")
-async def calculate_spectrum(payload: Payload):
+async def cal_spectrum(payload: Payload):
```

Comment:
```suggestion
async def calc_spectrum(payload: Payload):
```
This is a more common abbreviation for "calculate".